package.json

```diff
@@ -54,14 +57,18 @@
   "license": "MIT",
   "description": "Manage docker-compose from Node.js",
   "devDependencies": {
+    "@types/jest": "^24.0.15",
+    "@types/node": "^12.6.2",
+    "@typescript-eslint/eslint-plugin": "^1.12.0",
+    "@typescript-eslint/parser": "^1.12.0",
     "composefile": "^0.3.0",
     "dockerode": "^2.5.7",
     "eslint": "^5.6.0",
     "eslint-config-es": "^0.8.11",
     "eslint-plugin-extended": "^0.2.0",
```

Comment:
What do you think about `eslint-config-standard-with-typescript`?

Reply:
I kind of prefer standardjs styling, like removing semicolons and similar, but wanted to keep the current styling untouched (as it's just my preference, so I intended on using what already is there).

I'd say we should switch to a different style in a subsequent PR, maybe? :)

Reply:
Another PR for `eslint-config-standard-with-typescript` is fine.